README.md

```diff
@@ -106,44 +105,15 @@ memory.To start, in your environment with ray installed, run:
     Port: 10001
     ```
 
-9. In your Airflow DAG python file, you must include the following in your
-`default_args` dictionary:
 
-    ```python
-    from ray_provider.xcom.ray_backend import RayBackend
-    .
-    .
-    .
-    default_args = {
```

Comment:
@pgzmnk what would happen if multiple tasks fail? Would it write the state to s3 multiple times?